packages/x-date-pickers/src/internals/components/PickersPopper.tsx

```diff
@@ -337,6 +337,19 @@ export function PickersPopper(inProps: PickerPopperProps) {
   };
 
   const ActionBar = components?.ActionBar ?? PickersActionBar;
+  const actionBarProps = useSlotProps({
+    elementType: ActionBar,
+    externalSlotProps: componentsProps?.actionBar,
+    additionalProps: {
+      onAccept,
+      onClear,
+      onCancel,
+      onSetToday,
+      actions: [],
+    },
+    ownerState: { wrapperVariant: 'desktop' },
```

Comment:
On the legacy picker, I'm only passing the variant to the ownerState, that's really a tiny detail since those components are going away